controlplane/pkg/tests/nsmd_test_utils.go

```diff
@@ -323,18 +353,14 @@ func (impl *nsmdFullServerImpl) addFakeDataplane(dp_name string, dp_addr string)
 func (srv *nsmdFullServerImpl) registerFakeEndpoint(networkServiceName string, payload string, nse_address string) *registry.NSERegistration {
 	return srv.registerFakeEndpointWithName(networkServiceName, payload, nse_address, networkServiceName+"provider")
 }
-func (srv *nsmdFullServerImpl) registerFakeEndpointWithName(networkServiceName string, payload string, nse_address string, endpointname string) *registry.NSERegistration {
+func (srv *nsmdFullServerImpl) registerFakeEndpointWithName(networkServiceName string, payload string, nsmName string, endpointname string) *registry.NSERegistration {
```

Comment:
Nit: could we use `nsmgrName`, or maybe `mgrName` something that distinguish us from the more generic `nsm`.